CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* FE-221: Display info if a SmartGraph is opened in new Graph Viewer.
```

Comment:
we probably don't need this in the changelog as this PR is raised to another branch with an open PR